group_vars/all.yml.sample

```diff
@@ -387,7 +387,7 @@ dummy:
 
 ## Rados Gateway options
 #
-#radosgw_frontend_type: beast # For additional frontends see: http://docs.ceph.com/docs/nautilus/radosgw/frontends/
+#radosgw_frontend_type: beast # For additionnal frontends see: http://docs.ceph.com/docs/nautilus/radosgw/frontends/
```

Comment:
I think this is unintentional

Reply:
Yes, this file is automatically generated by the script, so we need to fix that